samples/flutter_app/lib/main.dart

```diff
@@ -74,13 +74,15 @@ class _ChatWidgetState extends State<ChatWidget> {
   final TextEditingController _textController = TextEditingController();
   final FocusNode _textFieldFocus = FocusNode();
   bool _loading = false;
+  late final String apiKey;
```

Comment:
We don't need to wait for `initState` to read a `const`. If you want a variable to  use the key multiple times, it can be static.
```suggestion
  static const _apiKey = const String.fromEnvironment('API_KEY');
```

Reply:
This is what happens when I write code at 7:30 am haha Thank You Nate!